src/pages/restore_account.vue

```
@@ -0,0 +1,268 @@
+const version = import.meta.env.VITE_APP_VERSION
+
+async function restoreAccount() {
+  dialogStore.openDialog({
+    id: 'restore-account-confirm',
+    title: t('are-you-sure-restore'),
+    buttons: [
+      {
+        text: t('button-restore'),
+        role: 'primary',
+        handler: async () => {
+          const supabaseClient = useSupabase()
```

ESLint

[error] 'supabaseClient' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)